live-build/hooks/20-extra-files.chroot

```diff
@@ -9,3 +9,9 @@ mkdir -p /lib/firmware
 mkdir -p /writable
 mkdir -p /var/lib/systemd/rfkill
 touch /etc/machine-id
+
+echo "creating fontconfig mount points" >&2
+mkdir -p /etc/fonts
```

Comment:
Note that this directory will be of little use because core's `/etc` is not, in general, used on classic systems.